src/Module.Server/CrpgSubModule.cs

```diff
@@ -16,6 +16,8 @@
 using TaleWorlds.Engine.InputSystem;
 
 using TaleWorlds.MountAndBlade.GameKeyCategory;
+using Crpg.Module.Common.ChatCommands;
```

Comment:
Should be under `#if CRPG_SERVER` preprocessor directive (L24)

Reply:
move